gui/app/templates/interactive/effect-options/custom-script.html

```diff
@@ -0,0 +1,20 @@
+<div class="effect-setting-container">
+    <div class="effect-specific-title"><h4>Which script would you like to use?</h4></div>
+    <div class="effect-info alert alert-info">
+        Place scripts in the <a id="scriptFolderBtn" href="#" ng-click="openScriptsFolder()">scripts folder</a> of the root Firebot directory, then refresh the dropdown.
+    </div>
+    <div class="btn-group">
+        <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            <span class="script-type">{{effect.scriptName ? effect.scriptName : 'Pick one'}}</span> <span class="caret"></span>
+        </button>
+        <a ng-click="getNewScripts()" id="refreshScriptList" href="#" style="padding-left:5px;height:100%;"><i class="fa fa-refresh" id="refreshIcon" style="margin-top:10px;" aria-hidden="true"></i></a>
+        <ul class="dropdown-menu script-dropdown">
+            <li ng-repeat="script in scriptArray" ng-click="effect.scriptName = script">
+                <a href>{{script}}</a>
+            </li>
```

Comment:
We could add a conditional \<li> element here when there is no scripts found, just for a better user experience. 

````
<li ng-show="scriptArray.length == 0" class="muted">
      <a href>No scripts found.</a>
</li>
````
"muted" is a css class I just added (so you may need to do a pull to see it), but it just sets the element to have an opacity of 0.5, giving it a softer look.